data/json/items/gunmod/stock.json

```diff
@@ -160,7 +160,7 @@
     "id": "stock_none",
     "type": "GUNMOD",
     "name": { "str": "sawn-off stock" },
-    "description": "A simple saw job cutting the original stock off this weapon.  Makes it much harder to handle, but much smaller.",
+    "description": "A simple saw job cut the original stock off this weapon.  It's now much harder to handle, but much smaller.",
```

Comment:
"A quick saw cut removed the original stock from this weapon. It is now much harder to handle, but much smaller.",